src/main/scala/io/ino/sbtpillar/Plugin.scala

```diff
@@ -89,7 +89,7 @@ object Plugin extends sbt.Plugin {
 
   def pillarSettings: Seq[sbt.Def.Setting[_]] = inConfig(Test)(taskSettings) ++ taskSettings
 
-  private case class CassandraUrl(hosts: Seq[String], port: Int, keyspace: String)
+  private case class CassandraUrl(hosts: Seq[String], port: Int, keyspace: String, var username: String = "cassandra", var password: String = "cassandra")
```

Comment:
Unfortunately the ```withCredentials``` call doesn't provide the same optional mechanism that the ```withQueryOptions``` provides. So I'd be forced to do something like:
```
val cluster = new Cluster.Builder()
          .addContactPointsSafe(url.hosts.toArray: _*)
          .withPort(url.port)
          .withQueryOptions(queryOptions)
url.credentials.foreach(cred => cluster.withCredentials(cred.username, cred.password)
cluster.build
```

If this looks cleaner to you, then I'll change it.